### PR TITLE
UrlScan - add reliability

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: demisto/etc#29347

## Description
Added reliability

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py [hunk 1]
```diff
@@ -26,9 +26,15 @@
 APIKEY = demisto.params().get('apikey')
 THRESHOLD = int(demisto.params().get('url_threshold', '1'))
 USE_SSL = not demisto.params().get('insecure', False)
+reliability = demisto.params().get('integrationReliability')
```
Comment (JasBeilin) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py:29]: Lets move this to main and use params =  demisto.params()

**Comment by JasBeilin** [comment]
Lets take another look at the tests and see if we can work something out. If not, we will add a picture of the instance with the change.

**Comment by yaakovpraisler** [comment]
Does not have tests.
image
image

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py [hunk 2]
```diff
@@ -22,29 +22,35 @@
...
-BASE_URL = 'https://urlscan.io/api/v1/'
-APIKEY = demisto.params().get('apikey')
-THRESHOLD = int(demisto.params().get('url_threshold', '1'))
-USE_SSL = not demisto.params().get('insecure', False)
 BLACKLISTED_URL_ERROR_MESSAGE = 'The submitted domain is on our blacklist. ' \
                                 'For your own safety we did not perform this scan...'
 
+
+class Client:
+    def __init__(self, api_key='', threshold=None, use_ssl=False, reliability='C - Fairly reliable'):
```
Comment (bakatzir) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py:30]: please use the enum for reliability

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py [hunk 3]
```diff
@@ -556,25 +563,45 @@ def format_http_transaction_list():
 
 
 def main():
+
+    params = demisto.params()
+
+    api_key = params.get('apikey')
+    threshold = int(params.get('url_threshold', '1'))
+    use_ssl = not params.get('insecure', False)
+    reliability = params.get('integrationReliability')
```
Comment (bakatzir) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py:572]:
```suggestion
    reliability = params.get('integrationReliability', DBotScoreReliability.C)
```

**Review by bakatzir — CHANGES_REQUESTED** [review]
Please make sure to actually test the reliability in the UT

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py (hunk 2 above)
Comment (yaakovpraisler) [reply Packs/UrlScan/Integrations/UrlScan/UrlScan.py]: done.

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py (hunk 3 above)
Comment (yaakovpraisler) [reply Packs/UrlScan/Integrations/UrlScan/UrlScan.py]: done.

**Review by bakatzir — APPROVED** [review]

**Pull request merged**